overlord/servicestate/quota_control.go

```diff
@@ -42,6 +50,54 @@ func checkSystemdVersion() {
 
 func init() {
 	checkSystemdVersion()
+	checkMemoryCGroupEnabled()
+}
+
+func checkMemoryCGroupEnabled() {
+	memoryCGroupError = memoryCGroupEnabled()
+}
+
+var CGroupsFilePath = "/proc/cgroups"
+
+//	since the control groups can be enabled/disabled without the kernel config the only
+//	way to identify the status of memory control groups is via /proc/cgroups
+//	"cat /proc/cgroups | grep memory" returns the active status of memory control group
+//	and the 3rd parameter is the status
+//	0 => false => disabled 1 => true => enabled
```

Comment:
this would be easier to parse on multiple lines
```suggestion
//	0 => false => disabled 
//	1 => true => enabled
```